src/main/java/com/google/javascript/gents/TypeConversionPass.java

```diff
@@ -208,8 +208,10 @@ public void visit(NodeTraversal t, Node n, Node parent) {
           if (declaration.rhs != null && declaration.rhs.isFunction()) {
             moveMethodsIntoClasses(declaration);
           } else {
-            // Ignore field declarations without a type annotation
-            if (declaration.jsDoc != null && declaration.jsDoc.getType() != null) {
+            // Ignore field declarations that are enums or classes
+            String comment = nodeComments.getComment(n);
+            if (!(declaration.rhs != null && declaration.rhs.isClass())
+                && (comment == null || comment.indexOf("@enum") < 0)) {
```

Comment:
Rather avoid ad-hoc searching for '@enum' here. I think `NodeUtil.getBestJSDocInfo(n).hasEnumParameterType()` should work here? Which reminds me, do we have a test for enums statics?